docs/src/routes/docs/components/date-picker.svx

```diff
@@ -0,0 +1,107 @@
+relatedComponents:
+  - name: Calendar
+    link: ./docs/components/calendar
+---
+
+<script>
+  import { DatePicker } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+</script>
+
+
+# DatePicker
+
+A date picker that supports both text input and a dropdown calendar.
+
+If you need to display the calendar differently, check out the [`Calendar`](./docs/components/calendar) component.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DatePicker format="%m/%d/%Y" />
+    <DatePicker format="%m/%d" noCalendar />
+    <DatePicker range />
+  </div>
+  <div slot="source">
+
+```svelte
+<DatePicker format="%m/%d/%Y" />
+<DatePicker format="%m/%d" noCalendar />
+<DatePicker range />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`noCalendar`** | `false` | `boolean` | Whether to hide the calendar dropdown and only keep the text input. |
+| **`top`** | `false` | `boolean` | Whether to show the calendar dropdown above or below the text field. |
+| **`right`** | `false` | `boolean` | Whether to align the calendar dropdown to the right or the left side of the text field. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`range`** | `false` | `boolean` | If this is set, the date picker will have two fields – for the start and the end of the date range. |
+| **`value`** | `null` | `Date` or `{ start, end }` | Depending on the value of the ==`range`== prop, this is either a Date object or an object with two fields, `start` and `date`, containing Date objects. |
+| **`locale`** | `undefined` | `string` | The language tag defining the desired locale (e.g., `en-US`). If left `undefined`, the user's locale will be used. |
+| **`firstWeekday`** | `1` | `number` | The index of the weekday to start the week at. <br /> 0 is for Sunday and 6 is for Saturday. |
+| **`format`** | `'%d.%m.%Y'` | `string` | The format string for the text input and representation. The `%`-specifiers are a subset of C date format specifiers, with only `%d`, `%m`, `%y` and `%Y` allowed. |
+
+### Class Properties
+
+All `<DatePicker>` components use the `.date-picker` class.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the component. Falsy values disregarded. |
+| **`weekdaysClass`** | `null` | `string` | A class string to the list of weekdays above the calendar. Falsy values disregarded. |
+| **`weekClass`** | `null` | `string` | A class string to add to the element containing each row of days in the calendar. Falsy values disregarded. |
```

Comment:
Do you mean "to each element containing a row of days" or the element containing all the rows? (I know it's most probably the former, but it can be made clearer)